lib/cli.js

```diff
@@ -3,37 +3,43 @@
 var esformatter = require('../lib/esformatter');
 var fs = require('fs');
 var glob = require('glob');
-var optimist = require('optimist');
-var stdin = require('stdin');
 var merge = require('mout/object/merge');
+var minimist = require('minimist');
 var options = require('./options');
-var EventEmitter = require('events').EventEmitter;
-
-// ---
-
-exports = module.exports = new EventEmitter();
-
-// ---
+var path = require('path');
+var stdin = require('stdin');
+var supportsColor = require('supports-color');
+
+var parseOpts = {
+  'boolean': [
+    'help',
+    'version',
+    'i',
+    'diff',
+    'diff-unified',
+    'no-color',
+    'color'
```

Comment:
These two don't seem to be used anywhere - leftover? If so, also drop them in `doc/cli.txt`.